lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -55,7 +33,7 @@ public function __construct() {
 	public function register_routes() {
 		register_rest_route(
 			$this->namespace,
-			'/' . $this->parent_base . '/(?P<font_family_id>[\d]+)/' . $this->rest_base,
+			'/' . $this->rest_base,
```

Comment:
By defining the `rest_base` as `font-families/(?P<font_family_id>[\d]+)/font-faces`, we can use this `rest_base` directly for registering the routes.